app.js

```diff
@@ -134,4 +134,8 @@ angular.module('PathOfDamage', [])
     });
   });
   $scope.hits.push(null);
+
+  window.onpopstate = function () {
+    window.location.reload();
+  }
```

Comment:
It's possible there is a slightly faster way to do this than refreshing the page, but I tried to implement the URL loading + Object key iteration as the onpopstate and it did weird stuff that I couldn't get to work.